numba/tests/test_tuples.py

```diff
@@ -772,6 +772,36 @@ def pyfunc(a):
 
         check((10, 20))
 
+    def test_untraced_value_tuple(self):
+        # This is a test for issue #6478.
+        def pyfunc():
+            a = (1.2, 1.3)
+            return a[0]
+
+        cfunc = jit(nopython=True, parallel=True)(pyfunc)
```

Comment:
Not all targets support `parallel=True`, so this is failing on the 32 bit architecture. You should mark all tests with `parallel=True` as `@skip_parfors_unsupported`